packages/service/tests/unit/test_pin_map_client.py

```diff
@@ -0,0 +1,80 @@
+"""Contains tests to validate the pin_map_client.py.
+"""
+
+import pathlib
+from typing import cast
+from unittest.mock import Mock
+
+import grpc
+import pytest
+from pytest_mock import MockerFixture
+
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.measurementlink.pinmap.v1.pin_map_service_pb2 import (
+    PinMap,
+    UpdatePinMapFromXmlRequest,
+)
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.measurementlink.pinmap.v1.pin_map_service_pb2_grpc import (
+    PinMapServiceStub,
+)
+from ni_measurement_plugin_sdk_service.discovery import DiscoveryClient
+from ni_measurement_plugin_sdk_service.grpc.channelpool import GrpcChannelPool
+from ni_measurement_plugin_sdk_service.pin_map._client import PinMapClient
+
+
+def test__valid_pin_map_file__register_pin_map___sends_request_and_returns_id(
```

Comment:
The "sends request" is the act and should not be quoted under the assert phrase.
```suggestion
def test___valid_pin_map_file___register_pin_map___returns_pin_map_id(
```